website/docs/enterprise/getting-started/install-enterprise.mdx

```diff
@@ -137,33 +119,828 @@ Your private GitHub repo should have a clusters/management folder that includes
+  --from-literal=tokenDuration=<token-duration> \
+  --from-literal=customScopes=custom,scopes
+```
+The format for the `customScopes` key is a comma-separated list of scopes to request. In this case, "custom", "scopes", and "openid" would be requested.
+
+#### Claims
+
+By default, the following claims are parsed from the OpenID [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#CodeIDToken) "email" and "groups". These are presented as the `user` and `groups` when WGE communicates with your Kubernetes API server.
+
+This is equivalent to [configuring](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#configuring-the-api-server) your `kube-apiserver` with `--oidc-username-claim=email --oidc-groups-claim=groups`.
 
-You do not need to install a CAPI provider to provision Kubernetes clusters using Weave GitOps Enterprise—you can also provision with Terraform. But for this example with CAPA, you must.
+Again, you can configure these from the `oidc-auth` `Secret`.
+
+```sh
+kubectl create secret generic oidc-auth \
+  --namespace flux-system \
+  --from-literal=issuerURL=<oidc-issuer-url> \
+  --from-literal=clientID=<client-id> \
+  --from-literal=clientSecret=<client-secret> \
+  --from-literal=redirectURL=<redirect-url> \
+  --from-literal=tokenDuration=<token-duration> \
+  --from-literal=claimUsername=sub \
+  --from-literal=claimGroups=groups
+```
+There are two separate configuration keys. You can override them separately. They should match your `kube-apiserver` configuration.
+
+</TabItem>
 
-Download a specific version of clusterctl from the [releases page](https://github.com/kubernetes-sigs/cluster-api/releases). We've tested the example templates provided in this guide with `clusterctl` version `1.1.3`.
+<TabItem value="Configuring OIDC with Dex and GitHub" label="Configuring OIDC with Dex and GitHub">
 
-Next, run this command:
+### Configuring OIDC with Dex and GitHub
+This example uses [Dex](https://dexidp.io/) and its [GitHub connector](https://dexidp.io/docs/connectors/github/) to show you how to log in to the Weave GitOps dashboard by authenticating with your GitHub account. It assumes you have already installed Weave GitOps on a Kubernetes cluster, per the instructions above, and have also [enabled TLS](#tls-configuration).
+
+Dex is an identity service that uses [OpenID Connect](https://openid.net/connect/) to
+drive authentication for other apps. There are other solutions for identity and access management, such as [Keycloak](https://www.keycloak.org/).
+
+Create a namespace where you will install Dex:
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dex
+```
+
+Get a GitHub ClientID and Client secret by creating a [new OAuth application](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+
+![GitHub OAuth configuration](/img/enterprise/getting-started/install-enterprise/github-oauth-application.png)
```

Comment:
shouldnt this be
```suggestion
![GitHub OAuth configuration](/img/guides/setting-up-dex/github-oauth-application.png)
```

```
Error: Image static/img/enterprise/getting-started/install-enterprise/github-oauth-application.png used in docs/enterprise/getting-started/install-enterprise.mdx not found.
```

Reply:
The thing is that #3919 hasn't been merged, yet, so if we merge this very one, the site won't build as does the "Docs / staging-release" CI check on this PR. Either we merge #3919 first or you change the base of one of these PRs to the branch of the other.

Reply:
@makkes Yes, I understand that. I was trying to keep PRs smaller while making multiple related changes at once. Makes sense to merge #3919 first.

Reply:
Ooops, didn't read through this, went and "fixed" the image links / emergency user links while testing it out locally. Will revert.